events/KWoC/kwoc.md

```diff
@@ -101,11 +112,21 @@ Share an **anonymus** feedback form with following
   - Verification link: A link of certificate hosted on a Public repository of KOSS - [kossiitkgp/public-files](https://github.com/kossiitkgp/public-files)
   - Stats link: A link containing the Student's or the Project's stats respectively for a student or a Mentor.
 
-- Move the website from `kwoc.kossiitkgp.org` to `kwoc{Y}.kossiitkgp.org` (e.g. `kwoc19.kossiitkgp.org` for KWoC19). Thereafter, the stats are hosted on the latter while the former is reused next year.
-
 - The certificates must be hosted on the public files repo - [kossiitkgp/public-files](https://github.com/kossiitkgp/public-files)
 - The certificates in PDF formats should be sent via mail to the participants.
 
+### KWoC Website Archival Guidelines
+
+- Move the website from `kwoc.kossiitkgp.org` to `kwoc{Y}.kossiitkgp.org` (e.g. `kwoc19.kossiitkgp.org` for KWoC19). Thereafter, the stats are hosted on the latter while the former is reused next year.
+
+- archived websites are branches in the frontend repo
+- these branches are deployed on [netlify](https://www.netlify.com/) because [vercel](https://vercel.com/) has a limitation on the number of branches.
+- `kwocxx.kossiitkgp.org` will be the domain of each archive
+- [`kwoc.kossiitkgp.org`](https://kwoc.kossiitkgp.org) will point to the previous KWoC's archive after it ends and before the next KWoC's website is deployed.
+
+> [!NOTE]
+> Each year a new website may be made or the previous one may be reused with suitable changes. Feel free to reuse whatever is required. 
```

Comment:
I would love to see a new design atleast every year. Makes the user's think that something has atleast changed from our end.